Eos/Fuego/EOS.H

```diff
@@ -265,9 +253,21 @@ void
 EY2G(amrex::Real E, amrex::Real Y[NUM_SPECIES], amrex::Real& G)
 {
   amrex::Real wbar, Cv, Cvx;
-  amrex::Real T = 300.;
+  amrex::Real T = 500.;
```

Comment:
Here again, we're picking a T and hoping the solver is robust and efficient enough to find the solution, but whenever we call this function, don't we have a decent guess to pass in for T?

Reply:
I completely agree. This was simply my attempt at keeping the `EY2G` function around. I think getting rid of `EY2G` altogether and make people rely on the `TY2G` function I added would make the most sense. Since gamma needs T and Y, I think we should require those to be passed in.